src/Core/Repositories/ISecretRepository.cs

```diff
@@ -4,8 +4,8 @@ namespace Bit.Core.Repositories
 {
     public interface ISecretRepository
     {
-        Task<IEnumerable<Secret>> GetManyByOrganizationIdAsync(Guid organizationId);
-        Task<Secret> GetByIdAsync(Guid id);
+        Task<IEnumerable<Secret>> GetManyByOrganizationIdAsync(Guid organizationId, bool includeDeleted = false);
+        Task<Secret> GetByIdAsync(Guid id, bool includeDeleted = false);
```

Comment:
I suspect we would want this to behave slightly different. You most likely want to fetch a list of only trash items, or only non trash items.

Reply:
resolved in https://github.com/bitwarden/server/pull/2144/commits/3481141d596c226572e6b0d893e70d716e0658c1